marketplace/extensions.json

```
@@ -547,4 +547,21 @@
 
     ]
   }
```

please add delimiter among the JSON objects

```suggestion
  },
```